index.html

```diff
@@ -1,51 +1,92 @@
-    <link rel="stylesheet" href="reset.css">
-    <link rel="stylesheet" href="style.css">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap" rel="stylesheet">
-    <title>My-Osom-Site</title>
-</head>
-<body>
-    <header>
-        <div class="nav">
-            <img src="logo.png" alt="logo">
-            <ul>
-                <li><a href="#">Home</a></li>
-                <li><a href="About/index.html" target="_blank">About</a></li>
-                <li><a href="Contact/index.html" target="_blank">Contact</a></li>
-            </ul>
-        </div>
-        <hr>
-    </header>
-    <main>
-        
-            <div class="hero">
-               <h1>My Osom Site</h1>
-            </div>
-       
-        <div class="blog">
-            <article class="article">
-                <h2>my first article</h2>
-                <p id="text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec id nibh sit amet tortor rhoncus pharetra non fermentum quam. Nam dui libero, ultricies a mi vitae, convallis congue felis. Aliquam erat volutpat. In sit amet faucibus tellus, vel semper justo. Fusce ullamcorper pharetra iaculis. Donec venenatis porta dolor, id viverra.</p>
-                <p id="date">20.02.2020</p>
-            </article>
-            <article class="article">
-                <h2>my second article</h2>
-                <p id="text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec id nibh sit amet tortor rhoncus pharetra non fermentum quam. Nam dui libero, ultricies a mi vitae, convallis congue felis. Aliquam erat volutpat. In sit amet faucibus tellus, vel semper justo. Fusce ullamcorper pharetra iaculis. Donec venenatis porta dolor, id viverra.</p>
-                <p id="date">24.04.2020</p>
-            </article>
-        </div>
-        <hr>
-    </main>
-    <footer>  
-        <p>Coded by: Wiktor Bonderek</p>
-        <p>wiktor.bonderek@shoper.pl</p>
-    </footer>
-</body>
+ <head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- <link rel="stylesheet" href="reset.css"> -->
+  <link rel="stylesheet" href="style.css">
+  <title>Lista zakupów</title>
+ </head>
+ <body>
+  
+   <header>
+     <hr>
+     <h1>Lista zakupów</h1>
+     <span>
+      <hr>
+      <hr>
+    </span>
+   </header>
+
+   <main>
+
+    <div class="wrapper">
+
+      <div class="fruits">
+        <div class="hero">
+          <h2>OWOCE</h2>
+        </div>
+        <div class="text-content">
+          <ul>
+            <li>Jabłka</li>
+            <li>Pomarańcze</li>
+            <li>Kiwi</li>
+          </ul>
+        </div>
+        <div class="checkbox">
+          <label><input type="checkbox" id="">Owoce kupione</label>
```

Comment:
id musi być unikalne per strone, napewno pusty string jest niedopuszczalny, fajnie by było jakbyś tutaj dodał name :) oraz oznaczył label atrybutem `for` z wartościa id która nie bd pusta :sweat_smile:

Reply:
dodatkowo imo powinn to trafić w tag form